portfolio/src/main/webapp/script.js

```diff
@@ -49,5 +50,19 @@ function createListComments(text){
     const liElement = document.createElement('li');
     liElement.innerText = text.comment;
 
+    const deleteButton = document.createElement('button');
+    deleteButton.innerText = 'Delete';
+    deleteButton.addEventListener('click',() => {
+        deleteComment(text);
```

Comment:
I stared at this for awhile wondering why passing in the text would work when your delete servlet requires the ID. I read most of the file before I realized that this is a comment object. Rename to `commentJson` to make it easier to figure out what's happening

Reply:
Oh I'm sorry, I didn't realize that, I'll change it, thank you for the feedback!